src/msw/thread.cpp

```diff
@@ -452,6 +453,9 @@ class wxThreadInternal
     // really start the thread (if it's not already dead)
     static THREAD_RETVAL DoThreadStart(wxThread *thread);
 
+    // really really start the thread, without catching exceptions
+    static THREAD_RETVAL DoDoThreadStart(wxThread *thread);
```

Comment:
Sorry but I really hate these "DoDo" names. Let's call it `DoThreadStartWithoutExceptionHandling()` or `DoThreadStartWithoutEH()` or even `DoThreadStartNoEH()` if it's too long.

Alternatively we could avoid having to come up with a name for it by using a lambda inside the function.